### PR TITLE
revise cli tool

### PR DESCRIPTION
**Pull request by sethvincent**
- renames flatsheet subcommands `account` and `sheet` to `accounts` and `sheets`
- adds callback function to flatsheet constructor
- now uses flatsheet constructor in the cli script instead of using accounts and sheets objects directly
- revises readme to reflect changes

### CODE REVIEW

File: bin/flatsheet [hunk 1]
```diff
@@ -57,27 +56,25 @@ function account () {
         }
       };
 
-      accounts.create(result.username, opts, function (err) {
-
-        //todo: notification of error
+      flatsheet.accounts.create(result.username, opts, function (err) {
         if (err) console.error(err);
-        db.close();
+        process.exit()
```
Comment (LukeSwart) [inline bin/flatsheet]: Is there a reason why we are using `process.exit()` instead of `db.close()`?

Comment (LukeSwart) [inline bin/flatsheet]: I think I get it. It doesn't seem like there is an easier way to close the db.

Comment (sethvincent) [inline bin/flatsheet]: aaahh, you're right. i was taking a shortcut with process.exit. pushing a commit that uses flatsheet.db.close instead

**Comment by LukeSwart** [comment]
I think it's good! If you approve of my minor changes commit, then let's merge it

**Comment by sethvincent** [comment]
awesome, thanks. merging.

**Pull request merged**